code/datums/weather/weather_types/snow_storm.dm

```diff
@@ -21,19 +21,18 @@
 	end_sound_vol = /datum/looping_sound/snowstorm::volume + 10
 
 	area_type = /area
-	protect_indoors = TRUE
```

Comment:
?

Reply:
It was converted to a `WEATHER_INDOORS` bitflag for the `weather_flags`

Reply:
the snowstorm datum doesn't use that flag is what I meant, sorry

Reply:
Ah I see. The bitflag was inverted which is why it's not included.

var/protect_indoors = TRUE // Weather can NOT affect indoor areas
weather_bitflag = WEATHER_INDOORS // Weather can affect indoor areas

Reply:
ah I see